client/src/models/glossary.js

```diff
@@ -26,7 +26,7 @@ const compiled_definitions = _.memoize((glossary_id) =>
   sanitized_marked(GlossaryEntry.lookup(glossary_id)._def_text)
 );
 
-const glossary_display = (item) => `<div>
+const glossary_display = (item) => `<div aria-live="polite">
```

Comment:
Good idea! I believe we still replace glossary tooltips with glossary links in the text-only site, but it's worth checking if this behaves well enough that we can switch to using tooltips on the text-only version too,